scopecheck/scopecheck.go

```diff
@@ -0,0 +1,99 @@
+package scopecheck
+
+import (
+	"fmt"
+
+	"github.com/taskcluster/taskcluster-cli/extpoints"
+	tcclient "github.com/taskcluster/taskcluster-client-go"
+	scopeutility "github.com/taskcluster/taskcluster-lib-scopes"
+)
+
+func init() {
+	extpoints.Register("scope-check", scopecheck{})
+}
+
+type scopecheck struct{}
+
+func (scopecheck) ConfigOptions() map[string]extpoints.ConfigOption {
+	return nil
+}
+
+func (scopecheck) Summary() string {
+	return "Shows whether a given scope satifies another."
+}
+
+func usage() string {
+	return `Usage:
+  taskcluster scope-check <scope1> satifies <scope2>
+`
+}
+
+func (scopecheck) Usage() string {
+	return usage()
+}
+
+type Auth tcclient.ConnectionData
+
+func New(credentials *tcclient.Credentials) *Auth {
+	myAuth := Auth(tcclient.ConnectionData{
+		Credentials:  credentials,
+		BaseURL:      "https://auth.taskcluster.net/v1",
+		Authenticate: true,
+	})
+	return &myAuth
+}
+
+//may have duplicate of this method because this method will be written at some point in issue#31
+func (myAuth *Auth) expandScope(payload *SetOfScopes) (*SetOfScopes, error) {
```

Comment:
Why not use:
https://godoc.org/github.com/taskcluster/taskcluster-client-go/auth#Auth.ExpandScopes